packages/rum-core/src/domain/rumEventsCollection/longTask/longTaskCollection.spec.ts

```diff
@@ -40,6 +44,16 @@ describe('long task collection', () => {
     expect(rawRumEvents.length).toBe(1)
   })
 
+  it('should only listen when the there is a session lite plan', () => {
+    const { lifeCycle, rawRumEvents } = setupBuilder.build()
+    session.setReplayPlan()
+    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, LONG_TASK)
+    session.setLitePlan()
+    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, LONG_TASK)
+
+    expect(rawRumEvents.length).toBe(1)
+  })
```

Comment:
```suggestion
  it('should only collect when session has a replay plan', () => {
    const { lifeCycle, rawRumEvents } = setupBuilder.build()
    
    session.setReplayPlan()
    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, LONG_TASK)
    expect(rawRumEvents.length).toBe(1)
    
    session.setLitePlan()
    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRY_COLLECTED, LONG_TASK)
    expect(rawRumEvents.length).toBe(1)
  })
```

not sure which one was generating the long task otherwise + test description